lib/sgcPrompt.js

```diff
@@ -26,11 +26,18 @@ const sgcPrompt = () => {
   }, {
     type: 'input',
     name: 'description',
-    message: 'Describe your commit message:',
+    message: 'Enter your commit message:',
+    validate: input => new Promise((resolve, reject) => {
+      if (input) {
+        resolve(true);
+      } else {
+        reject('A commit message is mandatory!');
+      }
+    }),
```

Comment:
Ah, okay, I was irritated by  `/* Preferred way: with promise */` in their docs. I'll change it.